railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -32,6 +32,21 @@ RUN curl -sL https://github.com/nodenv/node-build/archive/master.tar.gz | tar xz
     rm -rf /tmp/node-build-master
 
 <% end -%>
+
+<% if using_bun? && dockerfile_bun_version -%>
+ENV BUN_INSTALL=/usr/local/bun
+ENV PATH=/usr/local/bun/bin:$PATH
+ARG BUN_VERSION=<%= dockerfile_bun_version %>
+RUN curl -fsSL https://bun.sh/install | bash -s -- "${BUN_VERSION}"
```

Comment:
That helps but I think we should not install a new version of bun each time the container is built. We don't do with Ruby, or even with nodejs. We shouldn't do with bun. It seems we can't ask for any patch level, so let's by default generate with 1.0.1

Reply:
Okay @rafaelfranca I've updated this in https://github.com/rails/rails/pull/49241/commits/d234124dd79f5144af129110b768f5019d3ed659.